src/components/timeline/form/TimelineEditor/TimelineLayerEditor.svelte

```diff
@@ -172,7 +189,40 @@
         <ChevronDownIcon slot="icon" />
       </SearchableDropdown>
     {:else if isExternalEventLayer(layer)}
-      <SearchableDropdown
+      {@const filterCount = getExternalEventLayerFilterCount(layer)}
+      <ExternalEventFilterBuilder
```

Comment:
Might be missing some styling to make the filter builder button full width? 

<img width="439" height="394" alt="Image" src="https://github.com/user-attachments/assets/753e5c67-beb4-446c-be65-57d8a0018430" />